pkgs/development/python-modules/pyevmasm/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, future
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "pyevmasm";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "crytic";
+    repo = pname;
+    rev = version;
+    sha256 = "134q0z0dqzxzr0jw5jr98kp90kx2dl0qw9smykwxdgq555q1l6qa";
+  };
+
+  propagatedBuildInputs = [ future ];
+
+  checkPhase = "python tests/test_EVMAssembler.py";
```

Comment:
Would this run with ``pytestCheckHook``? I find it easier to use that than specifically running with ``python``